int8 TAKE_OFF = 1
int8 HOLD = 2
int8 MISSION_EXECUTION = 3
int8 AUTO_LAND = 4
int8 ONGROUND = 5
#request fields
int8 dummy
---
#response constants
int8 result
float64 remaining_estimate